arrow-optics/src/main/kotlin/arrow/optics/extensions/SetKAt.kt

```diff
@@ -2,15 +2,19 @@ package arrow.optics.extensions
 
 import arrow.core.SetK
 import arrow.core.k
-import arrow.extension
 import arrow.optics.Lens
 import arrow.optics.PLens
 import arrow.optics.typeclasses.At
 
 /**
  * [At] instance definition for [SetK].
  */
-@extension
+@Deprecated(
+  "Typeclass interface implementation will not be exposed directly anymore",
+  ReplaceWith(
+    "At<SetK<A>, A, Boolean>",
+    "arrow.optics.typeclasses.At", "arrow.core.SetK"),
+  DeprecationLevel.WARNING)
 interface SetKAt<A> : At<SetK<A>, A, Boolean> {
```

Comment:
☝️ 

Wondering the reason we can get rid of these interfaces now. Is it because we do not need @extension any longer?

Reply:
They're public so users might be using them directly.